rock-paper-scissors/service/src/main.rs

```diff
@@ -0,0 +1,136 @@
+use oasis_std::{Address, Context};
+
+#[derive(oasis_std::Service)]
+struct RockPaperScissors {
+    player_name: String,
+    player_move: u32, // TODO would be better as an enum?
```

Comment:
Example of how to do it as an enum: https://github.com/oasislabs/oasis-rs/blob/master/examples/messaging/src/main.rs#L71